java/vector/src/main/java/org/apache/arrow/vector/TypeLayout.java

```diff
@@ -136,6 +137,11 @@ public TypeLayout visit(Decimal type) {
         return newFixedWidthTypeLayout(BufferLayout.dataBuffer(128));
       }
 
+      @Override
+      public TypeLayout visit(FixedSizeBinary type) {
+        return newFixedWidthTypeLayout(BufferLayout.dataBuffer(type.getByteWidth() * 8));
```

Comment:
could you just do?
```
return newFixedWidthTypeLayout(new BufferLayout(BufferType.DATA, type.getByteWidth() * 8)
```

Then you don't need to `BufferLayout dataBuffer(int typeBitWidth)`
and then keep the

Reply:
fixed